src/Plugin.ts

```diff
@@ -17,6 +17,7 @@ export interface SaveOptions {
 export interface RecordOptions {
   content: boolean;
   excludePaths: string[];
+  hostPatterns: string[];
```

Comment:
I suggest renaming it to `includeHosts` to keep consistency through other similar fields (i.e. `excludePaths`)

Reply:
Done